src/lib/rest/StringFilter.cpp

```diff
@@ -1587,13 +1662,29 @@ bool StringFilter::qMatch(ContextElementResponse* cerP)
     {
       ContextAttribute* caP = cerP->contextElement.getAttribute(itemP->attributeName);
 
-      if ((itemP->op == SfopExists) && (caP == NULL))
```

Comment:
This changes on qMatch() are not really needed for the .test incluced in this PR? I understand that this is work "in advance" for the next PR, the one that will complete the functionality for notifications. Is my understanding correct?